src/components/popups/TeamSetting.tsx

```diff
@@ -206,6 +209,38 @@ const TeamSettings = ({ isOpen, onClose, onEdit, setIsLeave }) => {
     }
   };
 
+  const sendInvitationEmail = async () => {
+    const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+    if (!emailRegex.test(email)) {
+      setEmailError("Invalid email format");
+
+      return;
+    }
+
+    setEmailError("");
+    const requestBody = {
+      teamUUID: teamUUID,
+      receiverEmail: email,
+    };
+    try {
+      await api.post(
+        `/api/v1/teams/${teamId}/invitations`,
+        JSON.stringify(requestBody),
+        {
+          headers: {
+            Authorization: `${token}`,
+          },
+        }
+      );
+
+      setEmail("");
+      setEmailError(`The invitation has been sent to ${email} !`);
+    } catch (error) {
+      console.error("Failed to send email:", handleError(error));
+      setEmailError("Could not send email");
+    }
```

Comment:
handle different kind of input according to api specs

Reply:
Should be corrected